addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -220,7 +220,7 @@ def migration_invoice_moves(env):
     ids2 = env.cr.fetchall()
     if ids1 or ids2:
```

Comment:
I think you can remove the `RETURNING` part of both queries and don't store the IDs, as they are not used at all.

Reply:
Done!

Reply:
I think you haven't pushed this change

Reply:
Sorry, i forgot to make the push